presto-docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -167,7 +167,7 @@ Specifier Description
 ``%D``    Day of the month with English suffix (``0th``, ``1st``, ``2nd``, ``3rd``, ...)
 ``%d``    Day of the month, numeric (``00`` .. ``31``)
 ``%e``    Day of the month, numeric (``0`` .. ``31``)
-``%f``    Microseconds (``000000`` .. ``999999``)
+``%f``    Nanoseconds (``000000000`` .. ``999999999``)
```

Comment:
Nanoseconds is wrong. The accurate way to talk about it would be something like `fraction of second (6 to 9 digits, 000000 .. 999999, 0000000 .. 9999999, 00000000 .. 99999999, 000000000 .. 999999999)`. But this is probably way to wordy.

Additionally, it would be nice to point out that this is compatible with "microseconds" in MySQL for both parse and format
* For parsing, "microseconds" (`000000..999999` are all acceptable form of "fraction of seconds (6 to 9 digits)")
* For formatting, since internally we only store 6 digits, it will print 6 digits, which would be "microseconds".

But this just make things even more wordy.

 Any ideas, @martint?